.github/workflows/tracing.yml

```diff
@@ -14,32 +14,52 @@ concurrency:
 jobs:
   macos:
     runs-on: macos-latest
+    env:
+      DD_API_KEY: ${{ secrets.DD_API_KEY_CI_APP }}
     steps:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/node/setup
       - run: yarn install
-      - run: yarn test:trace:core:ci
+      - run: MOCHA_FILE=test-results/tracing/macos.xml yarn test:trace:core:ci
+      - run: |
+          curl -L --fail "https://github.com/DataDog/datadog-ci/releases/latest/download/datadog-ci_darwin-x64" --output "/usr/local/bin/datadog-ci"
+          chmod +x /usr/local/bin/datadog-ci
+          datadog-ci junit upload --service dd-trace-js-tests --tags os.platform:darwin --tags runtime.version:14 test-results/tracing/macos.xml
       - uses: codecov/codecov-action@v2
 
   ubuntu:
     runs-on: ubuntu-latest
+    env:
+      DD_API_KEY: ${{ secrets.DD_API_KEY_CI_APP }}
     steps:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/node/setup
       - run: yarn install
       - uses: ./.github/actions/node/14
-      - run: yarn test:trace:core:ci
+      - run: MOCHA_FILE=test-results/tracing/linux-14.xml yarn test:trace:core:ci
       - uses: ./.github/actions/node/16
-      - run: yarn test:trace:core:ci
+      - run: MOCHA_FILE=test-results/tracing/linux-16.xml yarn test:trace:core:ci
       - uses: ./.github/actions/node/latest
-      - run: yarn test:trace:core:ci
+      - run: MOCHA_FILE=test-results/tracing/linux-latest.xml yarn test:trace:core:ci
+      - run: |
+          curl -L --fail "https://github.com/DataDog/datadog-ci/releases/latest/download/datadog-ci_linux-x64" --output "/usr/local/bin/datadog-ci"
+          chmod +x /usr/local/bin/datadog-ci
+          datadog-ci junit upload --service dd-trace-js-tests --tags os.platform:linux --tags runtime.version:14 test-results/tracing/linux-14.xml
```

Comment:
This is repeated multiple times. Should it go directly in `yarn test:trace:ci`? Also, any way to standardize the file name in a way that won't need to be repeated? Alternatively, does `datadog-ci` support reporting all XMLs from a folder in one call?

Reply:
yes it does support that, but it does not support setting different tags to different reports. Notice that `runtime.version` is different for each command

Reply:
if we think having `runtime.version` as a tag is not critical I can pass `test-results/tracing` and call it a day